src/routes/posts.js

```diff
@@ -14,24 +14,24 @@ const router = express.Router()
 const createPost = async (req, res, next) => {
 	try {
 		const userId = req.user.id
-		const { title, image, content, published } = req.body
+		const { title, image, article, published } = req.body
```

Comment:
@beblicarl there'll be a bit more work to do aside this. This is because I got to understand the `posts.js` was meant to be a generic function for both `articles` and `gifs` hence the naming from the beginning. 

Now that we want articles to have their own response. It's time for a bit of refactoring and I will serve as a guide in this case.

are you up for it?

Reply:
yes please, i am up for it @jocrah

Reply:
okay so first, by trying to make `post.js` handle everything we've started getting unwanted routes. For example, as of now, an article can be created by using both `/articles` and `/feed` which was not the intention

First create a separate `articles.js` in the `src/routes`. and move the `createPost` function to that

Reply:
let me know when you're done so we can continue

Reply:
you can reach me on Slack too if anything isn't clear

Reply:
sure, i will do that